resources/init-postgres/create-db.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
What is this for? Why don't we bootstrap sample data with a init-Script in Docker? This way I am getting errors that my local user cannot be found.

Reply:
Strange. Did you run `docker-compose down --volumes`

Reply:
This file is used to setup the two databases. Another possible problem: The .env.example has changed slightly, you need to update your .env accordingly.